### PR TITLE
Add prettytable 0.7.2

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
Last commit to prettytable was in mid-2014.  Think it's worth it to ping the author and ask if they would like to be a maintainer?

File: recipes/prettytable/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+test:
+  imports:
+    - prettytable
+  commands:
+    - python {{ environ.SRC_DIR }}/prettytable_test.py
+
+about:
+  home: https://github.com/lmaurits/prettytable/
+  license: BSD License
```
Comment (jakirkham) [inline recipes/prettytable/meta.yaml]: Please specify number of clauses and drop "License".

File: recipes/prettytable/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - prettytable
+  commands:
+    - python {{ environ.SRC_DIR }}/prettytable_test.py
```
Comment (jakirkham) [inline recipes/prettytable/meta.yaml]: Please move this to the `script` section.

**Comment by pmlandwehr** [comment]
Whoops. See #878

**Comment by ericdill** [comment]
Works for me!

**Pull request closed without merging**